package.json

```diff
@@ -8,14 +8,15 @@
 		"build:assets": "copyfiles -u 1 \"src/**/*.sql\" \"src/**/.keep\" \"src/public/**\" dist",
 		"cleanup": "rimraf dist",
 		"format:prettier": "prettier . --write",
-		"format:sqlfluff": "sqlfluff fix src/database/",
+		"format:sqlfluff": "./venv/bin/sqlfluff fix src/database/",
 		"lint": "npm run lint:eslint && npm run lint:prettier && npm run lint:tsc && npm run lint:swagger",
 		"lint:eslint": "eslint ./src --ext .ts --max-warnings=0",
 		"lint:swagger": "swagger-spec-validator src/openapi.json",
 		"lint:tsc": "tsc --noEmit -p tsconfig.dev.json",
 		"lint:prettier": "prettier . --check",
-		"lint:sqlfluff": "sqlfluff lint src/database/",
+		"lint:sqlfluff": "./venv/bin/sqlfluff lint src/database/",
 		"migrate": "ts-node src/scripts/migrate.ts | pino-pretty",
+		"postinstall": "python3 -m venv venv && ./venv/bin/pip install -r sqlfluff/requirements.txt",
```

Comment:
I think that running the -m venv every time might be destructive, so I'm gonna tweak this to instead be:

```
[ ! -d \"venv\" ] && python3 -m venv venv;  ./venv/bin/pip install -r sqlfluff/requirements.txt"
```